app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -25,106 +26,83 @@ keywords:
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+### How bursaries are paid
 
-[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025).
+A bursary is paid over a minimum of 10 months by your teacher training provider across the duration of your course. They will confirm when you will get the payments. 
 
-### Scholarships
+You will not need to pay tax or National Insurance on bursary payments. 
 
-#### Eligibility
+### How to get a bursary
 
-To be eligible for a scholarship, you’ll need at least one of the following:
+You do not need to apply for a bursary. When you apply for teacher training, your provider will check if you're eligible for a bursary. If you are, your provider will confirm this in writing before you start your course. 
+
+## Eligibility for scholarships
+
+You must be eligible for student finance to receive a scholarship (unless you’re training to teach physics or languages). You’ll also need at least one of the following:
 
 * a bachelor’s degree class 2:1 (honours) or higher
 * a master’s degree
 * a PhD
 
-In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
+If you achieved a 2:2 in your degree, you may still be eligible. You would need to show you have enough relevant experience and subject knowledge. 
+
+Scholarships are awarded by independent organisations which set their own assessment requirements.
 
-#### Benefits
+Scholarships offer more than just financial benefits. For example, they can provide professional body membership, subject-specific support, mentoring, events and classroom resources. 
 
-Scholarships are provided by charitable organisations with a passion for their subject area.
+### How scholarships are paid
 
-They offer more than just financial benefits, including:
+A scholarship is paid over a minimum of 10 months by your teacher training provider across the duration of your course. They will confirm when you will get the payments. 
 
-* subject specialist events and webinars
-* online and face-to-face workshops
-* mentoring and coaching
-* a community of ex-scholars, teachers and subject leaders
-* access to classroom resources
-* networking events during your teacher training and beyond
+You will not need to pay tax or National Insurance on scholarship payments. 
 
-#### How to apply
+### How to apply for a scholarship
 
-Scholarships are offered by independent institutions. They set their own eligibility criteria and you’ll need to apply through the relevant scholarship body:
+You need to apply directly to the scholarship provider via their website. They will confirm if you meet their eligibility requirements. Each scholarship provider sets their own application deadline, which you can find on their website:
```

Comment:
very minor - I wonder if this para needs tweaking slightly

at the moment, it feels a little the main reason you'd look at the provider websites is to find out application deadlines, as that is when the bullets kick in

in reality, they have more information about the scholarships, the criteria, the application process as well as the deadline?

Reply:
```suggestion
You need to apply directly to the scholarship provider. Each provider sets their own application deadline and will confirm if you meet their eligibility requirements. You can find on about how to apply, deadlines and more details about each scholarship on the scholarship providers' websites:
```